src/components/Hashcat/Hashcat.tsx

```diff
@@ -1,32 +1,9 @@
-import { Button, LoadingOverlay, NativeSelect, NumberInput, Stack, TextInput, Grid, Alert } from "@mantine/core";
+import { Button, LoadingOverlay, NativeSelect, NumberInput, Stack, TextInput, Grid } from "@mantine/core";
 import { useForm } from "@mantine/form";
-import { IconAlertCircle } from "@tabler/icons";
 import { useCallback, useState } from "react";
```

Comment:
You forgot to add useEffect
```suggestion
import { useCallback, useState, useEffect } from "react";
```